tests/test_build.py

```diff
@@ -59,6 +59,10 @@ def test_build_book(file_regression, sphinx_build):
             f"https://datahub.berkeley.edu/hub/user-redirect/git-pull?repo=https://github.com/executablebooks/sphinx-book-theme&urlpath=lab/tree/sphinx-book-theme/TESTPATH/{path}"  # noqa E501
             in ntbk_text
         )
+        assert (
```

Comment:
I think that we can remove all of the `assert` statements now that you're using the regressions test

Reply:
done!